parsec/backend/cli/run.py

```diff
@@ -332,6 +332,12 @@ def handle_parse_result(self, ctx, opts, args):
 organization_id, device_id, device_label (can be null), human_email (can be null), human_label (can be null)
 """,
 )
+@click.option(
+    "--organization-default-users-limit",
```

Comment:
Sorry to be nerdy about naming, but I can't help myself ! 😄 

First, the name should contain the fact that we only count non-revoked users (so I guess `--organization-default-active-users-limit`)

Second, I don't think "default" is the good word here:
- it makes confusing to talk about the default value of the this parameter (i.e. there is no limit on active users if this variable is not configured)
- a "default" implied it could be overwritten. However this is something that can't be done during organization creation (the `organization_create` command doesn't have a field to change that, and the spontaneous bootstrap doesn't allow that for obvious reason ^^). So this option can be modified later on, but it initial value is always the one configured here.

Hence I would call it `--organization-initial-active-users-limit` ;-)

And this way we can have:
```python
    help="Non-revoked users limit used to configure newly created organizations (default: no limit)",
```